core/toolbox/collapsible_category.js

```diff
@@ -32,277 +31,277 @@ const {ToolboxSeparator} = goog.require('Blockly.ToolboxSeparator');
 
 /**
  * Class for a category in a toolbox that can be collapsed.
- * @param {!toolbox.CategoryInfo} categoryDef The information needed
- *     to create a category in the toolbox.
- * @param {!IToolbox} toolbox The parent toolbox for the category.
- * @param {ICollapsibleToolboxItem=} opt_parent The parent category or null if
- *     the category does not have a parent.
- * @constructor
  * @extends {ToolboxCategory}
```

Comment:
You don't need this extends either (and the one above still needs to be fixed as well)